docs/SNMP_Alarms.md

```diff
@@ -41,3 +41,19 @@ obtain the brief description, detailed description, and severity for the alarm:
 ## Alarm Models
 
 The alarm models used by Clearwater are defined in [alarmdefinition.h](https://github.com/Metaswitch/cpp-common/blob/master/include/alarmdefinition.h).
+
+## Alarm Resiliency
+
+Clearwater is designed so that if an EMS receiving alarm notifications fails and
+recovers, it will learn about any alarms raised by Clearwater while it was down,
+rather than those alarms just being lost. This is due to the following two
+features:
+
+*   The process responsible for sending SNMP notifications, the Alarm Agent,
+    keeps track of all the currently active error states in a table called the
+    Alarm Active Table. Upon restart, if supported, an EMS can read this table and
+    gain knowledge of any SNMP INFORMs it may have missed in its downtime.
+*   If an EMS does not support reading the Active Alarm Table we can still
+    recover the SNMP INFORMs by manually calling the synchronise alarms script.
```

Comment:
Instead of saying "by manually calling the synchronise alarms script" and then "To call this script use the command: `/usr/share/clearwater/bin/sync_alarm.py`", why not combine them? "...recover the SNMP INFORMs by running `/usr/share/clearwater/bin/sync_alarm.py`"